skema/program_analysis/CAST/matlab/tests/test_assignment.py

```diff
@@ -1,42 +1,35 @@
 from skema.program_analysis.CAST.matlab.matlab_to_cast import MatlabToCast
-from pathlib import Path
+from skema.program_analysis.CAST2FN.model.cast import (
+    Assignment,
+    LiteralValue,
+    Name,
+    Var
+)
 
 # Test the CAST returned by processing the simplest MATLAB assignment
 
-# The source code for the single line assignment test is 'x = 5'
-# The correct CAST for this single line assignment would be a single node: 
-#   node_type 'Assignment' 
-#   left.val.name 'x'
-#   right.value '5'
-#   right.value_type 'Integer'
-
 def test_assignment():
-    """ Tests whether the parser can handle a single assignment statement """
-    filepath = Path(__file__).parent / "data" / "assignment.m"
-    print(f"Testing parse tree for {str(filepath)}")
-    parser = MatlabToCast(filepath)
-    cast = parser.out_cast
+    """ Tests parser assignment CAST """
+
+    source = 'x = 5;'
```

Comment:
I think hardcoding the source like this is okay to do for the tests right now, I do effectively this in my test scripts, but I want to note that at some point we will be changing tests to access test files that are online on a google drive. I think we just haven't decided how to go about doing that yet.

Reply:
Sure, that's how I had been doing it.   I can easily switch back if needed.